lib/util/html.js

```diff
@@ -9,12 +9,24 @@
 //
 // Helper functions that relate to the HTML/frontend layer of the application.
 
+const { encodeNonAsciiHTML } = require('entities');
 const config = require('config');
 
 const envDomain = config.get('default.env.domain');
 
-// handy dev function for enabling syntax hilighting of html
-const html = ([ first, ...rest ], ...vars) => first + vars.map((v, idx) => [ v, rest[idx] ]).flat().join('');
```

Comment:
I don't 100% understand what this did before. It just concatenates everything, right? And the purpose is that your text editor or IDE recognizes <code>html``</code> and does syntax-highlighting?